feconf.py

```diff
@@ -505,6 +506,11 @@ def get_empty_ratings():
 
 # Unfinished features.
 SHOW_TRAINABLE_UNRESOLVED_ANSWERS = False
+# Number of unresolved answers to be displayed in the dashboard for each
+# exploration.
+TOP_UNRESOLVED_ANSWERS_COUNT_DASHBOARD = 3
```

Comment:
I can confirm that the latency wouldn't be worse than what's already in develop. And that's because the no. of datastore queries remains the same, it's just that we need a bit of extra computation of data this time!